src/main/java/echoServer/http/RequestParser.java

```diff
@@ -0,0 +1,14 @@
+package echoServer.HTTP;
+
+public class RequestParser {
+
+    public static HTTPRequest parser(String rawRequest) {
+
+        String[] rows = rawRequest.split("\r\n");
```

Comment:
You've been hard-coding in `"\r\n"` in a lot of places now. Consider making it a more generally available constant.